de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/project/filesystem/IntelliJResourceImpl.java

```diff
@@ -82,7 +82,7 @@ public IPath getProjectRelativePath() {
     }
 
     public SPath getSPath() {
-        return new SPath(project, getProjectRelativePath());
+        return null;//new SPath(project, getProjectRelativePath(), null);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=276&issues=AWajNu-jrChX7_AlOxX1&open=AWajNu-jrChX7_AlOxX1)

[ISSUE_KEY]: AWajNu-jrChX7_AlOxX1